bazelrc-preset.bzl

```diff
@@ -44,11 +38,13 @@ def _generate_preset(ctx):
     content.add("")
 
     for flag, meta in FLAGS.items():
-        if type(meta) == type([]):
-            for meta_item in meta:
-                content = _generate_preset_flag(content, flag, meta_item)
-        else:
-            content = _generate_preset_flag(content, flag, meta)
+        content.add("# {}".format(flag))
+        content.add("# Docs: https://registry.build/flag/bazel@{}?filter={}".format(version, flag))
+        if type(meta) != type([]):
+            meta = [meta]
+        for meta_item in meta:
+            _generate_preset_flag(content, flag, meta_item)
```

Comment:
```suggestion
            content = _generate_preset_flag(content, flag, meta_item)
```

In general, the changes look good to me, but I would keep the return value. Similar to Python, the return would not be required since all args are references, but I saw this pattern often in Python (numpy, pandas, or many other libraries design their interfaces like this) for better readability. Are there any reasons not to do this in starlark?

Reply:
only that we aren't consistent about using the return value.